### ISSUE TITLE
Support for Unreal 5

### ISSUE THREAD

**Issue by liveprints** [NinevaStudios/mqtt-utilities-unreal#26]
Hi there - the sources aren't sources but simply the plugin folder. I would like to build for Unreal 5, do you see any chance in this happening? Am happy to sponsor with 100 Euro... Especially if it were built by Friday...

**Comment by tustanivsky** [comment]
> Hi there - the sources aren't sources but simply the plugin folder. I would like to build for Unreal 5, do you see any chance in this happening? Am happy to sponsor with 100 Euro... Especially if it were built by Friday...

You can just change the `EngineVersion ` to `5.0.0` or `5.1.0` in [MqttUtilities.uplugin](https://github.com/NinevaStudios/mqtt-utilities-unreal/blob/master/MqttUtilities/MqttUtilities.uplugin) and rebuild the plugin. I recall it worked just fine with UE5 so no other special tweaks should be required

**Comment by samuelfalade** [comment]
I am not sure if he is referring to the error that pops up " The following modules are missing or built with a different engine version: MqttUtilities Would you like to rebuild them now?"

**Comment by tustanivsky** [comment]
> I am not sure if he is referring to the error that pops up " The following modules are missing or built with a different engine version: MqttUtilities Would you like to rebuild them now?"

Yeah, you'll need to rebuild those modules in order to use the plugin
